cmd/forwarder/run/run.go

```diff
@@ -43,7 +43,10 @@ type command struct {
 }
 
 func (c *command) RunE(cmd *cobra.Command, _ []string) error {
-	config := bind.DescribeFlags(cmd.Flags())
+	config, err := bind.DescribeFlags(cmd.Flags(), false, bind.JSON)
```

Comment:
How about we stick to the Plain format here?

Reply:
Oof, yes. Thanks for catching that.